setup/gcp/installer/forseti_server_installer.py

```diff
@@ -47,12 +49,22 @@ def __init__(self, **kwargs):
         """
         super(ForsetiServerInstaller, self).__init__()
         self.config = ServerConfig(**kwargs)
+        self.v1_config = None
 
     def preflight_checks(self):
         """Pre-flight checks for server instance"""
 
         super(ForsetiServerInstaller, self).preflight_checks()
-
+        _, zone, name = gcloud.get_forseti_v1_info()
+        if name is not None:
```

Comment:
Instead of name, can we call it `forseti_v1_name`?  Then it will be clear how it's being used in the code here, and you won't need the comment.

Reply:
Updated.